spec-main/guest-window-manager-spec.ts

```diff
@@ -197,7 +197,7 @@ describe('webContents.setWindowOpenHandler', () => {
 
   it('fires handler with correct params', async () => {
     const testFrameName = 'test-frame-name';
-    const testFeatures = 'top=10&left=10&something-unknown&show=no';
```

Comment:
was this just ... wrong before? or is this valid syntax which our parser doesn't correctly handle?

Reply:
I think it was just wrong before

Reply:
https://developer.mozilla.org/en-US/docs/Web/API/Window/open
> windowFeatures Optional
A string containing a comma-separated list of window features in the form name=value — or for boolean features, just name. These features include options such as the window's default size and position, whether or not to open a minimal popup window, and so forth. The following options are supported:
...